examples/protein_ligand/protein_ligand.ipynb

```diff
@@ -209,7 +213,11 @@
     "# This will take 1-2 minutes to load\n",
     "ff14sb = ForceField(\"ff14sb_off_impropers_0.0.2.offxml\")\n",
     "ff14sb.deregister_parameter_handler(\"ImproperTorsions\")\n",
```

Comment:
```suggestion
```
This can be dropped with version 0.0.2 - something I forgot to propagate out. You had no real way of knowing this but it caught my eye here.

Reply:
I tried this during the rewrite and it caused an error during export, but retrying it now it seems to work. Must've just accumulated one too many inconsistencies in that notebook kernel! Thanks.